package.json

```diff
@@ -11,12 +11,18 @@
   "author": "Raz Efron",
   "license": "ISC",
   "dependencies": {
+    "@googlemaps/google-maps-services-js": "^3.1.4",
+    "bcrypt": "^5.0.0",
+    "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "fs": "0.0.1-security",
     "https": "^1.0.0",
+    "jsonwebtoken": "^8.5.1",
     "mongoose": "^5.9.22",
+    "passport": "^0.4.1",
```

Comment:
Just to minimize risks, and since it doesn't provide and real value for us -
Let's remove all the `^` wildcards, to make sure that once we test the web app and deploy it, we know nothing's gonna change or fail because of a version update. It happened to me once and it's super hard to debug bugs that originate from a dependency breaking backwards compatibility.
This is its meaning: https://stackoverflow.com/a/22345808